conftest.py

```diff
@@ -38,6 +40,11 @@
 from mslib.mswms.demodata import DataFiles
 import tests.constants as constants
 
+matplotlib_logger = logging.getLogger('matplotlib')
+matplotlib_logger.setLevel(logging.INFO)
+mslib_logger = logging.getLogger('mslib')
```

Comment:
this now changes any of our logging entries to debug. I think we should only change the 'matplotlib' logging

Reply:
limit the code repetition by importing that, see above